src/manifest.ts

```
@@ -0,0 +1,61 @@
+    }],
+    host_permissions: ['https://github.com/*'],
+    web_accessible_resources: [
+      {
+        resources: ['assets/*'],
+        matches: ['<all_urls>'],
+      },
+    ],
+  }
+
+  if (isDev) {
+    delete manifest.content_scripts
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)